src/components/structures/GroupView.js

```diff
@@ -856,14 +888,15 @@ export default React.createClass({
                             width="18" height="18" alt={_t("Cancel")} />
                     </AccessibleButton>,
                 );
-                roomBody = <div>
-                    <textarea className="mx_GroupView_editLongDesc" value={this.state.profileForm.long_description}
+                bodyNodes = [
+                    <textarea className="mx_GroupView_editLongDesc"
+                        value={this.state.profileForm.long_description}
                         onChange={this._onLongDescChange}
                         tabIndex="3"
-                    />
-                    { this._getFeaturedRoomsNode() }
-                    { this._getFeaturedUsersNode() }
```

Comment:
So featured rooms / users are unused now? Should we remove the functions?

Reply:
I can't really justify keeping them. Although, we're going to need them Eventually...